### PR TITLE
Add the skip item for wistron 3200K platform

### PR DESCRIPTION
**Pull request by RogerX87**
### Description of PR

Summary:
Skip some platform tests due to HW design limitation

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
Avoid some unexpected failures when platform test
#### How did you do it?
Add skip case in tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml
#### How did you verify/test it?
Perform platform test cases
#### Any platform specific information?
N/A
#### Supported testbed topology if it's a new test case?
N/A

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml [hunk 1]
```diff
@@ -609,6 +627,18 @@ platform_tests/test_reboot.py:
     conditions:
       - "hwsku in ['Celestica-E1031-T48S4']"
 
+platform_tests/test_reboot.py::test_power_off_reboot:
+  skip:
+    reason: "Skip power off reboot test for Wistron"
+    conditions:
+      - "'sw_to3200k' in hwsku"
```
Comment (wangxin) [inline tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml:634]: The conditional mark is using "longest match" algorithm.

When we run test case `platform_tests/test_reboot.py::test_power_off_reboot`, the longest match would be this entry. The conditions defined in `platform_tests/test_reboot.py` will not be evaluated. This means that this test case will be executed on platform `Celestica-E1031-T48S4`. This is unexpected.

Need to change the condition string from 
```
      - "'sw_to3200k' in hwsku"
```
to
```
      - "hwsku in ['Celestica-E1031-T48S4'] or 'sw_to3200k' in hwsku"
```

File: tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml [hunk 2]
```diff
@@ -609,6 +627,18 @@ platform_tests/test_reboot.py:
...
+
+platform_tests/test_reboot.py::test_watchdog_reboot:
+  skip:
+    reason: "Skip watchdog reboot test for Wistron"
+    conditions:
+      - "'sw_to3200k' in hwsku"
```
Comment (wangxin) [inline tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml:640]: Similar issue here.

Comment (RogerX87) [reply tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml]: modified this according the suggestion you provided.

Thanks.

**Comment by WistronNetwork** [comment]
@wangxin , please help review, thanks a lot.

**Review by wangxin — APPROVED** [review]

**Comment by wangxin** [comment]
@WistronNetwork There is merge conflict. Can you do a rebase to the master branch and force push to your branch for this PR?

**Comment by RogerX87** [comment]
done the rebase and force push to update. Could you check it?

**Comment by RogerX87** [comment]
@wangxin please help to review

**Pull request merged**